stdlib/public/Concurrency/ContinuationProtocol.swift

```diff
@@ -0,0 +1,109 @@
+import Swift
+
+/// A value that can continue a suspended task.
+public protocol _Continuation {
```

Comment:
An underscored protocol is still an ABI liability, and we can't stop outside code from adding further conformers. It'd be better to duplicate the code.

Reply:
Alright. I strongly dislike repeating code but I defer to your expertise. Will change.